contentcuration/contentcuration/assets/channelList/index.js

```diff
@@ -0,0 +1,44 @@
+import Vue from 'vue';
+import VueRouter from 'vue-router';
+import Vuetify from 'vuetify';
+import ChannelListIndex from './views/ChannelListIndex.vue';
+import store from './store';
+import router from './router';
+
+import 'shared/i18n/setup';
+
+import 'vuetify/dist/vuetify.min.css';
+import 'shared/styles/main.less';
+
+Vue.use(VueRouter);
+Vue.use(Vuetify);
+
+Vue.filter('kindIcon', function(kind) {
+  switch (kind) {
```

Comment:
Note: I am working on a shared component for this: https://github.com/jayoshih/content-curation/blob/vue-edit/contentcuration/contentcuration/static/js/edit_channel/sharedComponents/ContentNodeIcon.vue